.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,15 +3,15 @@
 | Branch?       | master for features / 2.7 up to 4.0 for bug fixes <!-- see below -->
 | Bug fix?      | yes/no
 | New feature?  | yes/no <!-- don't forget to update src/**/CHANGELOG.md files -->
-| BC breaks?    | yes/no
+| BC breaks?    | no     <!-- see https://symfony.com/bc -->
 | Deprecations? | yes/no <!-- don't forget to update UPGRADE-*.md files -->
-| Tests pass?   | yes/no
-| Fixed tickets | #... <!-- #-prefixed issue number(s), if any -->
+| Tests pass?   | yes    <!-- please add some, will be required by reviewers -->
+| Fixed tickets | #...   <!-- #-prefixed issue number(s), if any -->
 | License       | MIT
-| Doc PR        | symfony/symfony-docs#... <!--highly recommended for new features-->
+| Doc PR        | symfony/symfony-docs#... <!-- will be required for new features -->
```

Comment:
I would say "is required for new features"

Reply:
updated